frontend/src/components/Carousel.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Wowowow, why do you want to disable an eslint rule ?
This rule specifies your imports should be functional, I don't see any case where it'd be a good idea to disable it